versioned_docs/version-0.4.2/tutorials/react-native.mdx

```diff
@@ -503,7 +525,30 @@ function ConnectScreen({ navigation }: ConnectScreenProps): JSX.Element {
 // ...
 ```
 
-Now the app is ready for the first test. If everything went well you should see
+Now the app is ready for the first test.
+
+This video shows how to connect to Jellyfish dashboard, create room and peers.
+
+<ImageWrapper url={"/react_native/dashboard_connect_to_media_server.gif"} />
+
+This image shows how you can obtain peer token.
+
+<ImageWrapper url={"/react_native/dashboard_peer_token.png"} />
+
+For physical devices, it would be easier to scan a QR code to place the peer token in the app. For a simulator/emulator, it is more convenient to copy the token to the clipboard and paste it in the simulator/emulator.
+:::note
+
+Don't connect with the same peer simultaneously from dashboard and mobile app.
+
+:::
+
+:::note
+
+Ios simulator does not support camera, but it will display remote tracks. It is strongly recommended to use physical device or android emulator where you can see your camera preview.
```

Comment:
```suggestion
iOS simulator does not support camera, but it will display remote tracks. It is strongly recommended to use physical device or Android emulator where you can see your camera preview.
```